src/Core/Form/IdentifiableObject/DataFormatter/BulkCombinationFormDataFormatter.php

```diff
@@ -42,33 +44,37 @@ class BulkCombinationFormDataFormatter extends AbstractFormDataFormatter
     public function format(array $formData): array
     {
         $pathAssociations = [
-            // References data follow same format so it is quite easy
             '[references][reference]' => '[references][reference]',
             '[references][mpn]' => '[references][mpn]',
             '[references][upc]' => '[references][upc]',
             '[references][ean_13]' => '[references][ean_13]',
             '[references][isbn]' => '[references][isbn]',
-            // Prices data have almost the same format
+            '[price][price_tax_included]' => '[price_impact][price_tax_included]',
             '[price][wholesale_price]' => '[price_impact][wholesale_price]',
             '[price][price_tax_excluded]' => '[price_impact][price_tax_excluded]',
-            '[price][price_tax_included]' => '[price_impact][price_tax_included]',
             '[price][unit_price]' => '[price_impact][unit_price]',
             '[price][weight]' => '[price_impact][weight]',
-            // Stock data are the trickiest
             '[stock][delta_quantity][delta]' => '[stock][quantities][delta_quantity][delta]',
             '[stock][fixed_quantity]' => '[stock][quantities][fixed_quantity]',
             '[stock][minimal_quantity]' => '[stock][quantities][minimal_quantity]',
             '[stock][stock_location]' => '[stock][options][stock_location]',
             '[stock][low_stock_threshold]' => '[stock][options][low_stock_threshold]',
-            '[stock][low_stock_alert]' => '[stock][options][low_stock_alert]',
+            sprintf(
```

Comment:
There's something wrong in the ay we handle `low_stock_threshold` it shouldn't be this complex I think there should be a way to simplify this if we remove the `low_stock_threshold` independent checkbox and rely fully on the  disabling switch feature instead

Reply:
As we discussed this may be worth a refacto for this specific field (including in the command and queries). But it will be done in a dedicated PR